samples/attested_tls/client/enc/Makefile

```diff
@@ -7,6 +7,7 @@ CFLAGS=$(shell pkg-config oeenclave-$(C_COMPILER) --cflags)
 CXXFLAGS=$(shell pkg-config oeenclave-$(CXX_COMPILER) --cflags)
 LDFLAGS=$(shell pkg-config oeenclave-$(CXX_COMPILER) --libs)
 INCDIR=$(shell pkg-config oeenclave-$(C_COMPILER) --variable=includedir)
+CRYPTO_LDFLAGS=$(shell pkg-config oeenclave-mbedtls_crypto --libs)
```

Comment:
Consider use variable here so you don't need to change this again when supporting OpenSSL version.